ctp2_code/ui/interface/greatlibrary.cpp

```diff
@@ -1747,25 +1762,15 @@ void GreatLibrary::UpdateList( DATABASE database )
 	m_topics_list->Clear();
 	m_listDatabase = database;
 
-	switch ( database ) {
-	case DATABASE_UNITS:
-
 #define HIDE(db, index) (db->Get(db->m_alphaToIndex[index])->GetGLHidden())
 
-		for (index = 0; index < g_theUnitDB->NumRecords(); index++)
```

Comment:
If you could make the variable index local here

Reply:
Done